### PR TITLE
Data Umbrella 2023 Review Blog

### PR DESCRIPTION
**Pull request by BerylKanali**

### CODE REVIEW

File: _posts/2024-01-02-du-2023-review.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+### Community Partnerships
+We are proud to have partnered with [PyGotham](https://pygotham.tv/), [PyData Global](https://pydata.org/global2023), [PyLadiesCon](https://conference.pyladies.com/), [NODES 2023](https://neo4j.com/event/nodes-2023/), [All ThingsOpen](https://2023.allthingsopen.org/), [WIA conference](https://www.dataconnectconf.com/), and [Diversify Tech](https://www.diversifytech.com/). Looking forward to awesome community partnerships in 2024. 
+
+### Website
+We are proud of our new and improved [website](https://www.dataumbrella.org/)! We hope you enjoy it.
+
+## Milestones
+### YouTube 
+We reached 3,000 subscribers on[YouTube](https://www.youtube.com/@DataUmbrella/)!
```
Comment (reshamas) [inline _posts/2024-01-02-du-2023-review.md:42]:
```suggestion
We reached 3,000 subscribers on [YouTube](https://www.youtube.com/@DataUmbrella/)!
```

File: _posts/2024-01-02-du-2023-review.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Data Umbrella was founded in December of 2019, and we are happy to have celebrated our 4th birthday!  Here we are sharing our accomplishments and acknowledging all the people who made it happen.
+
+## Highlight Accomplishments
+
+### Talks
+We are grateful to our esteemed speakers, the pioneers and experts who generously share their insights and expertise with our community. We also appreciate our community members for your active participation,contributions, and enthusiasm have created an environment for learning and collaboration.
+We had a total of 23 talks in 2023. All these talks are available on our [YouTube](https://www.youtube.com/@DataUmbrella/) channel.
+
+### Timestamps
+We have made significant progress in the [timestampsproject](https://github.com/data-umbrella/event-transcripts/issues/92) project. There are only 9 timestamps
```
Comment (reshamas) [inline _posts/2024-01-02-du-2023-review.md:21]:
```suggestion
We have made significant progress in the [timestamps](https://github.com/data-umbrella/event-transcripts/issues/92) project. There are only 9 timestamps. 
```

File: _posts/2024-01-02-du-2023-review.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+### YouTube 
+We reached 3,000 subscribers on[YouTube](https://www.youtube.com/@DataUmbrella/)!
+
+### Meetup 
+Our Meetup membership grew to:  
+- [DataUmbrella](https://www.meetup.com/data-umbrella/): 2808 members
+- [Data Umbrella Africa](https://www.meetup.com/data-umbrella-africa2/): 1066 members
+
+### Sponsors
+CZI and Google Open Source have been our sponsors for 2023.
```
Comment (reshamas) [inline _posts/2024-01-02-du-2023-review.md:50]:
```suggestion
CZI (Chan Zuckerberg Initiative) and Google Open Source have been our sponsors for 2023.
```

**Comment by BerylKanali** [comment]
Hi @reshamas 
I have made the necessary updates.

**Comment by reshamas** [comment]
@BerylKanali Looks great!! Thank you!

**Pull request merged**